package.json

```diff
@@ -20,5 +15,36 @@
   "homepage": "https://github.com/orangeui/orange#readme",
   "engines": {
     "node": ">=8.6"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack-dev-server --watch --mode development",
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode development",
+    "build:production": "webpack --mode production",
```

Comment:
@matevzgolavsek is this one needed since the basic `build` is the same production one?